### PR TITLE
Update CB to backup doc

### PR DESCRIPTION
**Pull request by mikedropmedia**
Add Pavithra comments

### CODE REVIEW

**Comment by mikedropmedia** [comment]
@cheng-tan ready for you

File: _guides/contextual_bandits.md [hunk 1]
```diff
@@ -10,113 +10,115 @@ body_class: tutorial
...
-Consider an application that interacts with its environment, such as a news website with users or a cloud controller with machines. Let's call this application _APP_. This application repeatedly goes through the following:
-1. Some context x arrives and is observed by _APP_
-2. _APP_ chooses an action _a_ from a set of actions _A_ i.e. a ∈ A to take (A may depend on x)
-3. Some reward r for the chosen a is observed by _APP_
+Bandit algorithms aid decision-making by making smarter choices in dynamic environments where options change rapidly, and the set of available actions is limited. Contextual Bandits are algorithms that use additional information (or context) to make better decisions when choosing actions.
+
+### Contextual Bandit example
 
-We want our application _APP_ to take actions such that we get the highest possible reward. In machine learning parlance, we want a _model_ that tells us which action to take.
+For this example, we use a hypothetical application called _APP_. This application interacts with the context of a user's behavior (search history, visited pages, or geolocation) in a dynamic environment–such as a news website or a cloud controller.
```
Comment (pavithraks) [inline _guides/contextual_bandits.md:29]: What is "this" referring to in "For this example..."

File: _guides/contextual_bandits.md [hunk 2]
```diff
@@ -10,113 +10,115 @@ body_class: tutorial
...
+_APP_ performs the following functions:
 
-In MAB, the gambler has no information at all to make a decision. However, our application _APP_ differs from MAB because we have some information available to the _APP_ which is the "context". Contextual Bandits uses additional information i.e. context available to make better decisions while choosing actions. Hence, the name "contextual" bandits.
+* Some context _x_ arrives and is observed by _APP_.
+* _APP_ chooses an action _a_ from a set of actions _A, i.e., a ∈A_ (_A_ may depend on _x_).
+* Some reward _r_ for the chosen a is observed by _APP_.
 
-In the contextual bandit problem, a learner (the gambler in the hypothetical experiment) repeatedly observes a context, chooses an action, and observes a loss/cost/reward for the chosen action only.
+Contextual Bandits use incoming data to help optimize a website to make better algorithmic decisions in real-time for the user.
```
Comment (pavithraks) [inline _guides/contextual_bandits.md:37]: I think using "incoming data" is a bit misleading. CB has an interactive set up. It observes a context, recommends an action and receives a reward. To just say "uses incoming data" wouldn't quite fully represent that.

**Comment by cheng-tan** [comment]
duplicate PR for #100, closing this one and all the comments need to addressed in #100.

**Pull request closed without merging**